src/Apps/Auction/Routes/__tests__/RegisterRoute.test.tsx

```diff
@@ -0,0 +1,71 @@
+// Begun from example in Register_analytics.test.tsx
+import { RegisterAppResponseFixture } from "Apps/__tests__/Fixtures/Auction/Routes/Register"
+import { createTestEnv } from "DevTools/createTestEnv"
+import { expectOne, RootTestPage } from "DevTools/RootTestPage"
+import { graphql } from "react-relay"
+import { flushPromiseQueue } from "Utils/flushPromiseQueue"
+import { RegisterRouteFragmentContainer } from "../Register"
+
+jest.unmock("react-relay")
+
+const { buildPage } = createTestEnv({
+  Component: RegisterRouteFragmentContainer,
+  defaultData: RegisterAppResponseFixture,
+  TestPage: class RegistrationTestPage extends RootTestPage {
+    get form() {
+      return expectOne(this.find("form"))
+    }
+  },
+  query: graphql`
+    query RegisterRoute_Test_Query {
+      sale(id: "whatever.") {
+        ...Register_sale
+      }
+      me {
+        ...Register_me
+      }
+    }
+  `,
+})
+
+xdescribe("Auction Registration App", () => {
+  // const getWrapper = (props: any) => {
```

Comment:
We haven't gotten _any_ of these tests working yet, so maybe we should not keep this file around.